src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -99,7 +99,7 @@ private static ErrorValue NotImplemented(IRContext irContext, [CallerMemberName]
         // - Error, 
         // - with updated values
         // Async because derived classes may back this with a network call. 
-        public virtual async Task<DValue<RecordValue>> AppendAsync(RecordValue record)
+        public virtual async Task<DValue<RecordValue>> AppendAsync(RecordValue record, CancellationToken cancel)
```

Comment:
>CancellationToken canc [](http://example.com/codeflow?start=87&length=22)

this is good 
-   tagging as a breaking change. I can help manage that.

Reply:
I went ahead and opened issue #655.